chapters/methodology.tex

```diff
@@ -29,3 +29,27 @@ \subsection{User Interface} \label{sec:user_interface}
 
 In terms of design, Android apps are generally expected to conform to Material Design, a set of guidelines defined by Google to help ensure both visual and practical consistency. As such, the user interface of the soHappy app is designed with Material Design in mind \cite{material_design}, incorporating principles such as animation or design based on the real world.
 
+\subsection{Model Training}
+
+As stated in the background section, soHappy uses a deep learning model to 
+provide information about a person smiling by utilizing TFLite.

```

Comment:
Personally, I would write "soHappy uses a deep learning model trained using TFLite" and leave "by utilizing TFLite" out.

Reply:
But thats wrong, the model was not trained by tflite.

Reply:
Sorry, didn't know that. You can disregard this comment.